cypress/support/commands/pageTestFragment/metaData.js

```diff
@@ -0,0 +1,197 @@
+Cypress.Commands.add('retrieveMetaDataContent', (metaDataTag, content) => {
+  cy.get(metaDataTag).should('have.attr', 'content', content);
+});
+
+Cypress.Commands.add('facebookMeta', (fbAdmins, appID, articleAuthor) => {
+  cy.get('head meta[name="fb:admins"]').should(
```

Comment:
The three cy.gets here should be chainable as they're all getting things from within head.